jest.config.cjs

```diff
@@ -53,7 +58,185 @@ const customJestConfig = {
+        functions: 100,
+        lines: 74,
+        statements: 74,
+      },
+    "src/app/(proper_react)/redesign/(authenticated)/user/welcome/GetStarted.tsx":
+      {
+        branches: 100,
+        functions: 50,
+        lines: 100,
+        statements: 100,
+      },
+    "src/app/(proper_react)/redesign/(authenticated)/user/welcome/View.tsx": {
+      branches: 95,
+      functions: 50,
+      lines: 99,
+      statements: 99,
+    },
+    "src/app/components/client/Chart.tsx": {
+      branches: 100,
+      functions: 25,
+      lines: 100,
+      statements: 100,
+    },
+    "src/app/components/client/ComboBox.tsx": {
+      branches: 83,
+      functions: 100,
+      lines: 100,
+      statements: 100,
+    },
+    "src/app/components/client/ExposureCard.tsx": {
+      branches: 73,
+      functions: 75,
+      lines: 79,
+      statements: 79,
+    },
+    "src/app/components/client/ExposuresFilter.tsx": {
+      branches: 75,
+      functions: 5,
+      lines: 73,
+      statements: 73,
+    },
+    "src/app/components/client/ExposuresFilterExplainer.tsx": {
+      branches: 100,
+      functions: 0,
+      lines: 16,
+      statements: 16,
+    },
+    "src/app/components/client/InputField.tsx": {
+      branches: 83,
+      functions: 100,
+      lines: 100,
+      statements: 100,
+    },
+    "src/app/components/client/ListBox.tsx": {
+      branches: 100,
+      functions: 0,
+      lines: 40,
+      statements: 40,
+    },
+    "src/app/components/client/LocationAutocompleteInput.tsx": {
+      branches: 62,
+      functions: 50,
+      lines: 74,
+      statements: 74,
+    },
+    "src/app/components/client/Popover.tsx": {
+      branches: 100,
+      functions: 0,
+      lines: 45,
+      statements: 45,
+    },
+    "src/app/components/client/dialog/Dialog.tsx": {
+      branches: 50,
+      functions: 50,
+      lines: 79,
+      statements: 79,
+    },
+    "src/app/components/client/toolbar/AppPicker.tsx": {
+      branches: 75,
+      functions: 37,
+      lines: 72,
+      statements: 72,
+    },
+    "src/app/components/server/Icons.tsx": {
+      branches: 90,
+      functions: 66,
+      lines: 68,
+      statements: 68,
+    },
+    "src/app/components/server/StatusPill.tsx": {
+      branches: 50,
+      functions: 100,
+      lines: 83,
+      statements: 83,
+    },
+    "src/app/functions/server/mockL10n.ts": {
+      branches: 91,
+      functions: 71,
+      lines: 92,
+      statements: 92,
+    },
+    "src/app/functions/universal/user.ts": {
+      branches: 100,
+      functions: 25,
+      lines: 68,
+      statements: 68,
+    },
+    "src/cloud-functions/index.js": {
+      branches: 63,
+      functions: 66,
+      lines: 60,
+      statements: 60,
+    },
+    "src/utils/*.js": {
+      branches: 19,
+      functions: 0,
+      lines: 0,
+      statements: 14,
+    },
+    "src/db/tables/*.js": {
+      branches: 0,
+      functions: 0,
+      lines: 0,
+      statements: 0,
+    },
+  },
```

Comment:
Ideally we never would - what would happen is that CI would start complaining if those new files/directories are not covered by unit tests :)

(One bit of nuance: the coverage report only measures files that actually get loaded during tests. So if not a single line of code in a file is imported when running the tests, it will not be counted as uncovered.)